connectors/serializers/csv.py

```diff
@@ -0,0 +1,13 @@
+import csv
+from typing import List, Tuple
+
+from connectors.serializers.base import BaseSerializer
+
+
+class CSVSerializer(BaseSerializer):
+    def serialize(self, rows: Tuple[List[str]], columns: List[str]):
+        with open(self.output_path, "w") as f:
+            writer = csv.writer(f)
+            writer.writerow(columns)
+            for row in rows:
+                writer.writerow(row)
```

Comment:
If the type of `row` is changed to allow things other than `str` (as per comment in `serializers/base`), this would need to be
```suggestion
                writer.writerow([str(val) for val in row])
```
since `csv` requires strings.